rules/windows/defense_evasion_disabling_windows_defender_powershell.toml

```diff
@@ -82,8 +82,11 @@ type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
- (process.name : ("powershell.exe", "pwsh.exe", "powershell_ise.exe") or process.pe.original_file_name in ("powershell.exe", "pwsh.dll", "powershell_ise.exe")) and
- process.args : "Set-MpPreference" and process.args : ("-Disable*", "Disabled", "NeverSend", "-Exclusion*")
+  (
+    process.name : ("powershell.exe", "pwsh.exe", "powershell_ise.exe") or
+    ?process.pe.original_file_name in ("powershell.exe", "pwsh.dll", "powershell_ise.exe")
```

Comment:
I barely ever see `in` used in Windows, due to case insensitivity. Of course, `pe.original_file_name` should always be the original file name, and if it's lowercase, `in` should be fine. Just pointing it out, that it might be a `:` candidate.

While continuing the review I noticed this in many rules, and I think it makes sense to use `in` and `==`. So will leave it here as consideration, but LGTM!